Spika/Data/Models/ServerModels/Message.swift

```diff
@@ -82,15 +83,15 @@ extension Message {
         case .text:
             return body?.text ?? " "
         case .image:
-            return "[Photo message]"
```

Comment:
Are the strings visible to User, is this localized?

Reply:
fixed